### ISSUE TITLE
[Question] Possible to define a standard tadpole-like (or curly) comma/quotation mark for Noto Sans series?

### ISSUE THREAD

**Issue by NightFurySL2001** [notofonts/latin-greek-cyrillic#411]
Currently, the design of Noto Sans uses a wedge-like shape for commas, semicolons, and quotation marks as follow. Most other languages under Noto Sans also follows this choice.
![image](https://user-images.githubusercontent.com/33471049/229278519-4776a985-72eb-45bb-a55a-1a4c881dbef0.png)

However, there are quite a few languages supported by Noto that included (or replaced) commas, semicolons, and quotation marks with a tadpole-like (or curly) style instead of the wedge style in Noto Sans (LCG). Here are a list of scripts that uses the tadpole-like comma and the closest match to other fonts (if applicable).
* Noto Sans Arabic
* Noto Sans CJK (copied/designed from Source Sans)
  * Noto Sans Yi (same as CJK fullwidth comma)
  * Noto Traditional Nüshu (same as CJK Latin comma)
* Noto Sans N'Ko (copied from Noto Naskh Arabic)
* Noto Sans Lisu
* Noto Sans Old Permic
![image](https://user-images.githubusercontent.com/33471049/229279126-c9f862fd-ec92-403b-97db-c3c646f115eb.png)

Question: If Noto Sans is to support a tadpole-like comma shape, which of these would be the designer's choice? Is it possible for the Noto team to decide on a sample glyph for the tadpole-like comma and apply it across all the aformentioned script fonts?

**Comment by fitojb** [comment]
> which of these would be the designer/Noto team's choice? 

I think you can ignore the Source Sans-derived shapes in Noto CJK, as that family is a fork of Source Han Sans

**Comment by NightFurySL2001** [comment]
Yeah I agree, and maybe not from N'Ko which came from Noto Naskh too. Personally I think the first from left in Noto Sans Arabic would be good, or the third.


________________________________
发件人: Adolfo Jayme-Barrientos ***@***.***>
发送时间: 2023年4月1日星期六 19:38
收件人: notofonts/latin-greek-cyrillic ***@***.***>
抄送: NFSL2001 ***@***.***>; Author ***@***.***>
主题: Re: [notofonts/latin-greek-cyrillic] [Question] Possible to define a standard tadpole-like (or curly) comma/quotation mark for Noto Sans series? (Issue #411)


which of these would be the designer/Noto team's choice?

I think you can ignore the Source Sans-derived shapes in Noto CJK, as that family is a fork of Source Han Sans

―

**Comment by moyogo** [comment]
The 9 and 6 forms for U+02BB, U+02BC, U+0313, U+0315 or reversed for U+2BD, U+0313 are much better considering the scope of the Noto fonts. Some languages need to have a clear distinction between l̓ and ĺ or m̓ and ḿ or even ʼ and ´ / ’.
The apostrophe-like or comma-like modifier letters and diacritics would be better if designed like the new Greek hard and soft spirits.

Related to #59, #111 and #186.

**Comment by verdy-p** [comment]
> Yeah I agree, and maybe not from N'Ko which came from Noto Naskh too. Personally I think the first from left in Noto Sans Arabic would be a good fit, or the fourth from left with slight modification.

I don't think that "Naskh" is a good source fitting for borrowing glyphs into N'Ko.

"Naskh" is designed with an oblique style for its diacritics (also slighly shifted to the left when above, or to the right when below), matching the way base letters are joining together (the Arabic Naskh diacritics strokes may also be slightly slanted about 30 degrees anti-clockwise compared to standard horizontal Arabic, independantly of the existence of an "italic" style applicable to both base letters and diacritics, plus some gylyph-to-glyph relative positioning for stacking glyphs may need to be rotated as they don't strictly align horizontally when side-by-side or vertically when on top of each other).

And this does not apply to N'Ko which should then be based on standard Arabic glyphs, with its horizontal joining and alignment for letters and vertical alignment for stocking on top of each other.

Other scripts to consider are: Coptic, Deseret, Cherokee, Lisu, modern Georgian (Mkhedruli/Mtavruli), plus Maths symbols and CJK fixed-width variants. They all have letters also derived or borrowed directly from Latin/Greek/Cyrillic shapes, with just minor adaptations of glyph metrics.

**Comment by NightFurySL2001** [comment]
Sadly, it seems that during Noto Sans Arabic v2.011 release, to match with other Arabic fonts, U+066B and its derivatives are redesigned. Only U+2E41 and U+060C is kept (which might be changed in a future version).
![image](https://github.com/notofonts/latin-greek-cyrillic/assets/33471049/e8bf08a2-fcb4-4512-b268-af36cc172f66)

I personally still like this U+066B style, mainly due to the horizontal cut and straight tail matching the current comma in Noto Sans LCG. It just might need a bit of adjustment to match the rectangular-like full stop.
![image](https://github.com/notofonts/latin-greek-cyrillic/assets/33471049/d8cd6c99-3561-4feb-844d-85db2688a318)

However if cross-script design is required, then U+2E41 might be better due to the curved tail. It is however debatable if straight tail vs curved tail will make any differences; a horizontally-terminated end might help more here.
![image](https://github.com/notofonts/latin-greek-cyrillic/assets/33471049/dfd509a7-a09c-438e-8c90-46e5d1a1c929)

**Comment by NightFurySL2001** [comment]
Maybe we can add a SS04 for tadpole-like comma for cases where it is required? Related glyphs like semicolon, combining commas and quotation marks can also be swapped in so that we can differentiate it with cedilla/other wedge symbols. I prefer this comma from old Noto Sans Arabic, maybe need some edits to match the full stop:
![image](https://github.com/notofonts/latin-greek-cyrillic/assets/33471049/d8cd6c99-3561-4feb-844d-85db2688a318)

**Comment by simoncozens** [comment]
I'm not convinced there's a case for this. Noto Sans is not trying to be an infinitely customizable font so that everyone can have their preferred design choices. Adding glyphs is not free. If you want a different design, other fonts are available. If you want Noto Sans with a different comma shape, it's an OFL font so knock yourself out.
